### ISSUE TITLE
French language formating error

### ISSUE THREAD

**Issue by amakh** [charliekassel/vuejs-datepicker#102]
Hello,

First, thanks for this component, it's really nice !

Using french language, the month of December seems to be formatted with the day abbreviation from the letter "D".
Issue is only for french language I think because of the month's accent "Décembre".

I quickly changed the demo link to show you the result : [demo here](http://www.webpackbin.com/EkbvXJl9f)  
`12 Lunécembre 2016` instead of `12 Décembre 2016`

(Lun is the abbreviation of Lundi -> Monday in english)

**Comment by itsDamien** [comment]
+1

**Comment by charliekassel** [comment]
I agree. The formatDate method is really very poor and not robust at all.
It needs a much better implementation, or integration with a library like moment.

**Comment by amakh** [comment]
Is there any quick fix to prevent this from happening?

**Comment by gfauconnet** [comment]
Look here for quick fix.
https://github.com/gfauconnet/vuejs-datepicker/commit/53c502e5fd790eac167eaa95183acfbe19d59711

I will do another pull request with this commit

**Comment by itsDamien** [comment]
It's already fixed in your PR #103 @gfauconnet

**Comment by gfauconnet** [comment]
Oh, it's my first pull request, sorry :)

**Comment by charliekassel** [comment]
@gfauconnet - if you make a separate PR for the dateUtils commit I am happy to merge that.

**Comment by adenvt** [comment]
Same error on language="**id**", month of **May** output `5ei` instead of `Mei`
